rxfingerprint/src/main/java/com/mtramin/rxfingerprint/CipherProvider.java

```diff
@@ -94,7 +94,7 @@ Cipher getCipherForEncryption() throws IOException, GeneralSecurityException {
 	@TargetApi(Build.VERSION_CODES.M)
 	Exception mapCipherFinalOperationException(Exception e) {
 		boolean shouldThrowKeyPermanentlyInvalidatedException = invalidatedByBiometricEnrollment &&
-				Build.VERSION.SDK_INT >= 26 /*Build.VERSION_CODES.O*/ &&
+				Build.VERSION.SDK_INT == 26 /*Build.VERSION_CODES.O*/ &&
```

Comment:
Just wondering, why did you hardcode the `26` instead of using the `Build.VERSION_CODES.O`?

Reply:
To use `Build.VERSION_CODES.O`, compileSdk (dependencies.gradle) must be set to 26 and I did not want to change dependencies.gradle file.